### ISSUE TITLE
Textures misaligned on rotated meshes

### ISSUE THREAD

**Issue by codecat** [codecat/godot-tbloader#9]
I noticed that some faces will have misaligned textures for meshes that are slightly rotated. I'm unsure what's the cause of this yet.

**Comment by fire** [comment]
Can you post a test project?

**Comment by codecat** [comment]
Definitely: [TBLoaderTextureBug.zip](https://github.com/codecat/godot-tbloader/files/9382667/TBLoaderTextureBug.zip)

Left = Godot, right = TrenchBroom
![image](https://user-images.githubusercontent.com/136534/185633977-a9499c24-3527-4127-9b09-e0b0cf18ae91.png)
